app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
||= hàm này có tác dụng gì em

Reply:
toán tử "||=" sẽ gán giá trị cho biến ở vế trái nếu biến đó trả về giá trị nil hoặc false. Vì vậy hàm này sẽ trả về đối tượng current_user đc tìm theo id nếu biến @current_user đang nil, để tăng hiệu năng, mỗi lần gọi đến hàm này thì User sẽ chỉ truy cập vào DB tìm đối tượng khi mà @current_user còn đang nil hoy